chart.tf

```diff
@@ -47,7 +48,7 @@ module "release" {
     },
     {
       name  = "config.MONGODB_HOST"
-      value = "${var.mongodb_host}"
+      value = jsondecode(data.aws_secretsmanager_secret_version.secrets.secret_string)["mongodb_host"]
```

Comment:
can we have this kind of lines refactored to alow data passed from both ${var.mongodb_host} and jsondecode(data.aws_secretsmanager_secret_version.secrets.secret_string)["mongodb_host"] ?

Reply:
It is possible if you tell me by what rules what value should be

Reply:
let have by default used credentials from variables (${var.mongodb_host} and etc.), and if non empty secret_id passed then get credentials from secret manager

Reply:
Done